Util/Docker/Release.Dockerfile

```diff
@@ -1,19 +1,28 @@
 
```

Comment:
Okay, this file feels like you basically removed a lot, but I don't know why.

Reply:
Yeah, but Release.Docker is obsolete now because the output of Carla.Dockerfile-mod is already the final image. This multi-stage build with the runtime stuff is including what Release.Docker used to do in one concise Dockerfile.